src/table/utils/handle-accessibility.js

```diff
@@ -34,9 +34,10 @@ export const removeAndFocus = (newCoord, rootElement, setFocusedCellCoord, keybo
   keyboard.enabled && keyboard.focus();
 };
 
-export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard) => {
+export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard, totalsPosition) => {
   const { rawRowIdx, rawColIdx } = cell;
-  removeAndFocus([rawRowIdx + 1, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
+  const firstBodyRowIdx = totalsPosition === 'top' ? 2 : 1;
```

Comment:
the variable name `firstBodyRowIdx` does looks confused to me.

Reply:
it is the index that the first row has, when you take header and totals into account. the `rawRowIdx` is the index in the body, then we need to add 1 or 2 to compensate. Do you have any other suggestions?

Reply:
how about
```
const cellRowIdx = totalsPosition === 'top' ? rawRowIdx + 2 : rawRowIdx + 1;
  removeAndFocus([cellRowIdx, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
```

Reply:
sure, I might call it `adjustedRowIdx` though